compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -177,9 +178,11 @@ std::set<std::string> gather_customcode_set(const ::tflchef::ModelRecipe &model_
   std::set<std::string> customcode_set;
   for (const auto &operation : model_recipe.operation())
   {
-    auto op_chef = op_chef_registry().lookup(operation.type()).create(&operation);
-    if (op_chef->code() == tflite::BuiltinOperator_CUSTOM)
-      customcode_set.insert(operation.type());
+    if ((operation.has_extype() && operation.extype() == "Custom") || operation.type() == "Custom")
+    {
+      assert(operation.custom_code() != "");
```

Comment:
This one would be more readable.
```suggestion
      assert(not operation.custom_code().empty());
```